mve_intrinsics/mve.md

```diff
@@ -7,23 +8,23 @@ Date of Issue: 02 July 2021
 
 ------------------------------------------------------------------------
 
-## Preface                                                                  
+# Preface                                                                  
```

Comment:
Please submit a separate PR for the headers change.

Reply:
Should I cherry-pick the commit into a separate branch and then revert the changes in each file accordingly for each branch?